internal/page/certificateprovider/confirm_your_details.go

```diff
@@ -48,6 +49,11 @@ func ConfirmYourDetails(tmpl template.Template, lpaStoreResolvingService LpaStor
 			App:                 appData,
 			CertificateProvider: certificateProvider,
 			Lpa:                 lpa,
+			PhoneNumberLabel:    "mobileNumber",
+		}
+
+		if lpa.IsPaperDonor {
+			data.PhoneNumberLabel = "contactNumber"
```

Comment:
There is already some label switching in the template. I think it would be good to either have all the label logic in here, or all in the template (and then maybe we decide that we will stick to that option going forward?)

Reply:
:+1: that's fine logic for me